app/services/search/google_search_service.rb

```diff
@@ -4,26 +4,19 @@ module Search
   class GoogleSearchService
     GOOGLE_SEARCH_URL = 'https://www.google.com/search?q=%{keyword}&hl=%{language}&lr=%{language}'
 
-    USER_AGENT_HEADER = 'User-Agent'
-
-    def initialize(keyword, language = 'en')
+    def initialize(keyword:, language: 'en')
       @search_url = format(GOOGLE_SEARCH_URL, keyword: keyword, language: language)
     end
 
     def search
       uri = URI.parse(@search_url)
 
-      response = Faraday.get(uri, nil, USER_AGENT_HEADER => generate_user_agent)
+      headers = {}.merge(UserAgentHeaderGenerator.call)
```

Comment:
Since we only have user-agent in the header, should we get rid of `{}.merge`? 

```suggestion
headers = UserAgentHeaderGenerator.call
```